### ISSUE TITLE
Make "speed up" and "cancel" more visible

### ISSUE THREAD

**Issue by bdresser** [MetaMask/metamask-extension#5374]
4.12 moves the "speed up" button from the extension home screen into the new transaction detail modal. While the modal is awesome, we've received intermittent user feedback that the "speed up" feature needs to be super clear

See
https://github.com/MetaMask/metamask-extension/issues/5075
https://github.com/MetaMask/Design/issues/25#issuecomment-414361081

Might be worth considering moving the "speed up" option back to the home screen, probably with more descriptive langague. ("Taking too long? Click here to speed up.")

Let's see what feedback we get from this initial change and regroup.

**Comment by leafcutterant** [comment]
If I speed up once, the button disappears. I can still speed the transaction up further, but have to look for the link on the transaction's history.

I propose keeping the button (and the link in the tx's history as well) regardless of whether it has been sped up before. Perhaps the button label could be changed to "Speed up more" if it has been sped up at least once.

**Comment by brad-decker** [comment]
@leafcutterant the transaction list redesign is coming in the much anticipated MetaMask v8 release, once it lands I would love it if you could give the new UI a spin and open a new issue if you find anything outstanding. I don't believe we have moved to allow multiple speed up attempts yet but the UI does make it easier to find and clears up some confusion. 

Thank you for your suggestion! If, after using v8, you still feel strongly about this please open up a new issue so it can be tracked independently of overall visibility of actions!

**Comment by brad-decker** [comment]
Closed via #8564
